### PR TITLE
Update tile to be compatibe for use with MS SQL

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: pattern-2/bosh-release/create.sh [hunk 1]
```diff
@@ -162,11 +169,13 @@ cd ..
 echo "---> Adding blobs..."
 
 bosh -e vbox add-blob ${distributions}/${jdk_distribution} openjdk/${jdk_distribution}
-bosh -e vbox add-blob ${distributions}/${mysql_driver} mysqldriver/${mysql_driver}
 bosh -e vbox add-blob ${distributions}/${wso2_product_pack_identifier}.zip ${wso2_product}/${wso2_product_pack_identifier}.zip
 bosh -e vbox add-blob ${distributions}/${wso2_product_keymanager_pack_identifier}.zip wso2is_km/${wso2_product_keymanager_pack_identifier}.zip
 bosh -e vbox add-blob ${distributions}/${wso2_product_analytics_pack_identifier}.zip ${wso2_product}_analytics/${wso2_product_analytics_pack_identifier}.zip
```
Comment (VimukthiPerera) [inline pattern-2/bosh-release/create.sh]: Shall we remove this unnecessary empty line?

File: pattern-2/bosh-release/jobs/health_check/spec [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: health_check
+
+templates:
+    run.sh: bin/run
+
+packages: []
+
+properties: {}
```
Comment (VimukthiPerera) [inline pattern-2/bosh-release/jobs/health_check/spec]: Shall we add an extra empty line at the end of the file?

File: pattern-2/bosh-release/jobs/health_check/templates/run.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    else
+        echo "WSO2 APIM $2 is not alive" >>/dev/stderr
+        exit 1
+    fi
+}
+
+carbonHealthCheckEP="https://localhost:9443/carbon/admin/login.jsp"
+
+health_check ${carbonHealthCheckEP} Carbon
+exit 0
```
Comment (VimukthiPerera) [inline pattern-2/bosh-release/jobs/health_check/templates/run.sh]: Missing empty line at the end of the file.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: pattern-2/tile/tile.yml [hunk 4]
```diff
@@ -31,12 +31,25 @@ forms:
...
+  - name: carbon_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+    - name: postgres
```
Comment (VimukthiPerera) [inline pattern-2/tile/tile.yml]: We do not support this at the moment. Shall we remove this for the moment?

File: pattern-2/tile/tile.yml [hunk 5]
```diff
@@ -31,12 +31,25 @@ forms:
...
+      label: postgres
+    - name: oracle.jdbc.driver.OracleDriver
```
Comment (VimukthiPerera) [inline pattern-2/tile/tile.yml]: Shall we remove this for the moment as well?

File: pattern-2/tile/tile.yml [hunk 6]
```diff
@@ -47,12 +60,25 @@ forms:
...
```
Comment (VimukthiPerera) [inline pattern-2/tile/tile.yml]: If someone selects this, it won't work as the bosh deployment does not support this for the moment. Shall we remove this?

File: pattern-2/tile/tile.yml [hunk 7]
```diff
@@ -63,12 +89,25 @@ forms:
...
```
Comment (VimukthiPerera) [inline pattern-2/tile/tile.yml]: Shall we remove this as well? We can add it when bosh deployment supports it.

File: pattern-2/bosh-release/create.sh (hunk 1 above)
Comment (Savidude) [reply pattern-2/bosh-release/create.sh]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/bosh-release/jobs/health_check/spec (hunk 2 above)
Comment (Savidude) [reply pattern-2/bosh-release/jobs/health_check/spec]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/bosh-release/jobs/health_check/templates/run.sh (hunk 3 above)
Comment (Savidude) [reply pattern-2/bosh-release/jobs/health_check/templates/run.sh]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/tile/tile.yml (hunk 4 above)
Comment (Savidude) [reply pattern-2/tile/tile.yml]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/tile/tile.yml (hunk 5 above)
Comment (Savidude) [reply pattern-2/tile/tile.yml]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/tile/tile.yml (hunk 6 above)
Comment (Savidude) [reply pattern-2/tile/tile.yml]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/tile/tile.yml (hunk 7 above)
Comment (Savidude) [reply pattern-2/tile/tile.yml]: fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c

File: pattern-2/bosh-release/config/blobs.yml [hunk 8]
```diff
@@ -1,20 +1,28 @@
+jdbcdrivers/mssql-jdbc-7.0.0.jre8.jar:
```
Comment (VimukthiPerera) [inline pattern-2/bosh-release/config/blobs.yml]: Shall we commit an empty file when sending a PR? IMO, updating this with every PR does not make sense.

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**